easybuild/easyconfigs/k/kalign2/kalign2-2.0.4-GCCcore-10.2.0.eb

```diff
@@ -0,0 +1,32 @@
+# Contribution from the NIHR Biomedical Research Centre
```

Comment:
You forgot to remove the old files when you switched names

Reply:
Silly me. One day I learn that! One day!
Thanks. I have removed them.